frontend/src/component/feature/FeatureView/FeatureOverview/ReleasePlan/ReleasePlan.tsx

```diff
@@ -247,8 +247,8 @@ export const ReleasePlan = ({
                         Release plan
                     </StyledHeaderTitleLabel>
                     <span>{name}</span>
-                    <StyledHeaderDescription>
-                        {description}
+                    <StyledHeaderDescription title={description}>
+                        <Truncator lines={2}>{description}</Truncator>
```

Comment:
Why only 2 lines not 3?

Reply:
We decided on 2 in the meeting I think, can always be changed later :)

Reply:
Happy to see the Truncator being used 😄 You should be able to set the title in the Truncator instead of int he StyledHeaderDescription. The behavior should be that the tooltip is only shown if the content is truncated.